detector/src/detector.cpp

```diff
@@ -0,0 +1,99 @@
+#include <detector/detector.hpp>
+#include <iostream>
+
+namespace ariitk::detector {
+
+Detector::Detector() {
+}
+
+Detector::~Detector() {
+    cv::destroyAllWindows();
+}
+
+void Detector::setHSVMin(const int& h , const int& s , const int& v) {
+    hsv_min_ = cv::Scalar(h,s,v);
+}
+
+void Detector::setHSVMax(const int& h , const int& s , const int& v) {
+    hsv_max_ = cv::Scalar(h,s,v);
+}
```

Comment:
all these mini sized functions (except ```setCannyParams``` isn't mini), put them directly in the header. this will keep the cpp cleaner